JetStreamDriver.js

```diff
@@ -280,30 +284,30 @@ class Driver {
 
         categoryScores = new Map;
         for (const benchmark of this.benchmarks) {
-            for (let category of Object.keys(benchmark.subTimes()))
+            for (let category of Object.keys(benchmark.subScores()))
                 categoryScores.set(category, []);
         }
 
         for (const benchmark of this.benchmarks) {
-            for (let [category, value] of Object.entries(benchmark.subTimes())) {
+            for (let [category, value] of Object.entries(benchmark.subScores())) {
                 const arr = categoryScores.get(category);
                 arr.push(value);
             }
         }
 
         if (isInBrowser) {
             summaryElement.classList.add('done');
-            summaryElement.innerHTML = "<div class=\"score\">" + uiFriendlyNumber(geomean(allScores)) + "</div><label>Score</label>";
+            summaryElement.innerHTML = "<div class=\"score\">" + uiFriendlyScore(geomean(allScores)) + "</div><label>Score</label>";
             summaryElement.onclick = displayCategoryScores;
             if (showScoreDetails)
                 displayCategoryScores();
             statusElement.innerHTML = '';
         } else if (!dumpJSONResults) {
             console.log("\n");
             for (let [category, scores] of categoryScores)
-                console.log(`${category}: ${uiFriendlyNumber(geomean(scores))}`);
+                console.log(`${category}: ${uiFriendlyScore(geomean(scores))}`);
```

Comment:
Can we do `${category}: ${uiFriendlyScore(geomean(scores))}, ${uiFriendlyDuration(geomean(times))}`?